CMakeLists.txt

```diff
@@ -7,11 +7,12 @@ set(CMAKE_AUTOUIC ON)
 set(CMAKE_AUTORCC ON)
 set(CMAKE_AUTOMOC ON)
 
-find_package(Qt5 COMPONENTS Core)
+find_package(QT NAMES Qt6 Qt5 COMPONENTS Core REQUIRED)
+find_package(Qt${QT_VERSION_MAJOR} COMPONENTS Core REQUIRED)
```

Comment:
This seems somewhat redundant...?

Reply:
I agree it _looks_ redundant, but it isn't. These two lines came from code generated by Qt Creator when you create a new empty project. If you remove either of them, you get CMake errors. I don't pretend to understand CMake — I'm used to qmake — but my guess is that the first one gets hold of the Qt5 & Qt6 Core components and the second one actually puts the right one into the mix.